src/components/AddedFeature.js

```diff
@@ -1,13 +1,24 @@
 import React from 'react';
-
+import {removeFeature} from '../actions/Features'
+import {connect} from 'react-redux';
 const AddedFeature = props => {
+  const removeItem = e =>{
+    e.preventDefault();
+    props.removeFeature(props.feature)
+   }
+
   return (
     <li>
       {/* Add an onClick to run a function to remove a feature */}
-      <button className="button">X</button>
+      <button className="button" onClick = {removeItem}>X</button>
```

Comment:
In JSX (and HTML) you shouldn't put spaces around your `=`
For example this should be
```jsx
onClick={removeItem}
```

This is in contrast to JavaScript where you should put spaces around your `=`
```js
const example = 0;
```